review/fileToReview.js

```diff
@@ -1,73 +0,0 @@
-
-var domain = "bank.local.fr"
```

Comment:
could be a constant, also it would be better to extract this in a configuration file